ming_sim/cli/terminal.py

```
@@ -191,13 +191,106 @@ def _skill_ids_from_text(session: GameSession, text: str) -> List[str]:
+                        session.db.capture_chat_rollback_snapshot(),
+                    )
+                session.db.fail_chat_turn(int(chat_turn_id))
+            else:
+                # Prior Q&A turn must stay intact; only drop the failed exit placeholder.
+                session.db.conn.execute(
+                    "DELETE FROM story_ledger_entries WHERE id = ?",
+                    (int(entry_id),),
+                )
+                session.db.conn.commit()
+        except BaseException:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)